src/results.jl

```diff
@@ -5,29 +5,29 @@ Given a MYSQL type get the corresponding julia type.
 """
 function mysql_get_julia_type(mysqltype)
     if (mysqltype == MYSQL_TYPE_BIT)
-        return Cuchar
+        return Union{Missings.Missing, Cuchar}
```

Comment:
Shouldn't it only return a ``Union`` if the column actually supports ``NULL``? If a column doesn't, this seems not right?

Reply:
There was some discussions of this in the slack -- I was concerned that the user not knowing in advance if they were going to get the union type or not might be problematic, but there wasn't a clear consensus.

Reply:
My comments may have been confusing on that regard: I agree that if it's possible to know if a column will have no nulls, then it's better to just return a normal Vector. Feather.jl does this since the # of nulls in a column is stored in the column's metadata. In ODBC, however, there's not enough metadata to know if a column will have nulls, so by default, every column is returned as `Union{T, Missing}`.

Reply:
I'd say there are two different situations: does the column allow for nulls vs. does it contain nulls. I think the type should not change depending on the latter, but if there exists a mechanism in MySQL to say that a column does not support nulls, it should be used to choose the type.

Reply:
There is `is_nullables` that gives an array describing whether columns accept nulls: see [here](https://github.com/JuliaDB/MySQL.jl/blob/724baf40c7b63437a6738714b6b9708f0fa70dd4/src/types.jl#L173)

Reply:
OK, so it definitely makes sense to map this property to the Julia column type.